contracts/floaty/Cargo.toml

```diff
@@ -35,7 +35,7 @@ cosmwasm-schema = { path = "../../packages/schema" }
 cosmwasm-std = { path = "../../packages/std" }
 schemars = "0.8.3"
 serde = { version = "1.0.103", default-features = false, features = ["derive"] }
-thiserror = "1.0.26"
+rand_chacha = "0.3.1"
```

Comment:
I'm surprised this does not cause issues with the getrandom dependency this pulls in. Don't know why it works. Usually you need to deactivate default features for those crates to only use the deterministic part.

Reply:
Oh, you're right. That's very weird. I'll disable default features anyways just to be sure.